tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -274,10 +274,18 @@ var CESIUM_MAP_VIEW = (function() {
         }
 
         for (let model of m_models) {
-            // options is where the model data is store.
-            let model_option = cesium_options(model.options);
+            var model_option;
+            // if model has options attribute, it should come from MVLayer.
+            if (model.options) {
```

Comment:
A better check for MVLayer is the source attribute. I'd change the check to something like:
```
if (model.source && model.source === 'CesiumModel')
```